.github/ISSUE_TEMPLATE/new-package.md

```diff
@@ -0,0 +1,28 @@
+---
+name: New package
+about: Suggest software for inclusion in the repo
+title: Package <name>
+labels: package
+assignees: ''
+
+---
+
+<!--
+Please fill out all the fields below if possible!
+Make sure the software
+1. does not already exist in the official Arch Linux repos: https://archlinux.org/packages/
+2. does not already exist in this repo: https://arch.osamc.de/#packages
+3. is released under a free software license
+-->
```

Comment:
Another requirement would be that the software can be built from source on x86_64, but it's probably to difficult for many requesters to check. Just as a side note.

Reply:
93e66cbdd6dd6428d27e81344a699d361c5da285

Reply:
I'd prefer "can be successfully built" instead of "successfully compiles". Not evry software needs compilation, sometimes a build involves more than compilation.

Reply:
3618177ea224523c8c6d50468f2dfa78750609b3